dts/bindings/led/dac-leds.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2018, Linaro Limited
+# Copyright (c) 2025, Prevas A/S
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Group of DAC-controlled LEDs.
+
+  Each LED is defined in a child node of the dac-leds node.
+
+  Here is an example which defines an LED in the node /leds:
+
+  / {
+    leds {
+      compatible = "dac-leds";
+      led_0 {
+        dac = <&dac1>;
+        channel = <0>;
+        resolution = <12>;
+      };
+    };
+  };
+
+  Above:
+
+  - led_0 uses dac1 channel 0 with 12 bit resolution.
+
+compatible: "dac-leds"
+
+child-binding:
+  description: DAC LED child node
+  properties:
```

Comment:
I would add a max value here at minimum, possibly even a max and forward current. If we are using a DAC to drive an LED, we should do it right!
![image](https://github.com/user-attachments/assets/09660062-74b5-4050-8d3b-3391a7797b45)

So add something like
```
max-voltage-mv:
  type: int
  description: max voltage in millivolt
forward-voltage-mv:
  type: int
  description: led forward voltage in millivolt
```
This way, the brightness percentage could be converted to a relative intensity quite nicely!

Reply:
My intention was to use this with an opamp circuit. E.g. [using TI's OPA310 example](https://www.ti.com/lit/ds/symlink/opa310.pdf#page=2).
This is not clear at all and should be clarified.

I like your suggestion. Maybe it could be added on top of this driver and bindings so both is possible?

Reply:
ok, that is a bit of a different story, usually LEDs are tied directly to the pin, with no voltage controlled current source circuit in between. Maybe add a boolean property named `linear` or `current-controlled` and `voltage-controlled` something? so the driver knows what "type" of led driving circuit (if any) is used?

Reply:
The reason this is not the case for PWM controlled LEDs BTW is that they are "fully on" / "fully off", so the current is not used to manage intensity :)

Reply:
sounds good :)

Reply:
I reworked the driver.